interface/themes/core/oe-mobile.scss

```diff
@@ -0,0 +1,34 @@
+/* customized settings to make OpenEMR mobile friendly*/
+/*for Smartphones*/
+@media only screen and (max-width: 400px) {
+    h1{
+        font-size: 30px;
```

Comment:
Does the system have a global font-size definition at body?  Can we use rem here for each of these h* tags?  Just a thought as it makes it easier to override from the body and everything naturally adapts.  Its a more fundamental question for all the stylesheets but it helps avoid hard-coding pixel sizes here.